### PR TITLE
v2 documentation link fix

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

**Comment by gyuho** [comment]
@xiang90 @heyitsanthony

File: Documentation/op-guide/runtime-configuration.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ It is recommended to enable this option. However, it is disabled by default beca
 [disaster recovery]: admin_guide.md#disaster-recovery
 [fault tolerance table]: admin_guide.md#fault-tolerance-table
 [majority failure]: #restart-cluster-from-majority-failure
-[member-api]: members_api.md
+[member-api]: ../v2/members_api.md
```
Comment (xiang90) [inline Documentation/op-guide/runtime-configuration.md]: probably we should update to use gRPC version of member api. Add a todo here?

Comment (gyuho) [inline Documentation/op-guide/runtime-configuration.md]: Ok just added TODO to `runtime-configuration.md`. Thanks.

**Comment by xiang90** [comment]
LGTM. Defer to @heyitsanthony

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**